scene/animation/animation_blend_tree.cpp

```diff
@@ -178,7 +178,7 @@ double AnimationNodeAnimation::_process(const AnimationMixer::PlaybackInfo p_pla
 	}
 	set_parameter(time, cur_time);
 
-	return is_looping ? HUGE_LENGTH : anim_size - cur_time;
```

Comment:
We explained in https://github.com/godotengine/godot/pull/82089 that this change is currently inacceptable. This revision is being worked on as https://github.com/godotengine/godot-proposals/issues/8083.

Reply:
To be fair, I **absolutelly** agree animationtree/node needs reworking, this PR is basically a makedo with what I got.

It's worth mentioning though, kinda like SkeletonIK3D, while reworking is mandatory, we do need something to work with when possible, after reading the proposal I understand my PR may cause more trouble than benefits, but I do hope we get something to work with in the meantime while animationtree gets reworked